gm4_mountaineering/data/gm4_mountaineering/functions/ski_effects/skis_equipped.mcfunction

```diff
@@ -0,0 +1,32 @@
+# Run all ski-related functions
+# @s = player wearing skis
+# at @s
+# run from player_main
+
+tag @s add gm4_mountaineering_using_skis
+
+# Adds tags for whether the player is skiing normally or skiing on a non-snowy block
+function gm4_mountaineering:ski_effects/check_snowy_block
+
+# Adds particles at the player's feet when skiing on a snowy bock and without air below
+execute if entity @s[tag=gm4_mountaineering_skiing] unless block ~ ~-2 ~ minecraft:air run particle minecraft:cloud ~ ~ ~ 0.1 0.1 0.1 0.001 50 normal
```

Comment:
particles feel way too much, try reducing count to like 2-3. Maybe a different particle altogether. It should feel more like the running particles